cli/remote_build.go

```diff
@@ -54,6 +55,7 @@ type RemoteCmd struct {
 	committed      bool
 	publicRepo     bool
 	remotes        []GitRemote
+	skipped        []string
```

Comment:
Can we avoid saving this list in the executable struct? We should largely use this structure for the command itself (flags / config primarily)